### ISSUE TITLE
NAOT dwarfdump has more warnings than expected

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/runtime#87321]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=300882
Build error leg or test failing: nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh
Pull request: https://github.com/dotnet/runtime/pull/87292

## Error Message

```json
{
  "ErrorMessage": "Found 20158 warnings and errors, expected between 12000 and 20000",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[300842](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300842)|dotnet/runtime|[nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300842&view=ms.vss-test-web.build-test-results-tab&runId=6097068&resultId=100007)|dotnet/runtime#86813|
|[300836](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300836)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300836&view=ms.vss-test-web.build-test-results-tab&runId=6096402&resultId=100023)|dotnet/runtime#87302|
|[300906](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300906)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300906&view=ms.vss-test-web.build-test-results-tab&runId=6095394&resultId=100023)|dotnet/runtime#87126|
|[300900](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300900)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300900&view=ms.vss-test-web.build-test-results-tab&runId=6095286&resultId=100023)|dotnet/runtime#86932|
|[300882](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300882)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300882&view=ms.vss-test-web.build-test-results-tab&runId=6095120&resultId=100023)|dotnet/runtime#87292|
|[300754](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300754)|dotnet/runtime|[nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300754&view=ms.vss-test-web.build-test-results-tab&runId=6091728&resultId=100017)|dotnet/runtime#86803|
|[300749](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300749)|dotnet/runtime|[nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300749&view=ms.vss-test-web.build-test-results-tab&runId=6091658&resultId=100011)|dotnet/runtime#86668|
|[300596](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300596)|dotnet/runtime|[nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300596&view=ms.vss-test-web.build-test-results-tab&runId=6088032&resultId=100015)|dotnet/runtime#71426|
|[300531](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300531)|dotnet/runtime|[nativeaot/SmokeTests/DwarfDump/DwarfDump/DwarfDump.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300531&view=ms.vss-test-web.build-test-results-tab&runId=6086716&resultId=100006)|dotnet/runtime#83855|
|[300499](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300499)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300499&view=ms.vss-test-web.build-test-results-tab&runId=6086352&resultId=100023)|dotnet/runtime#87292|
|[300493](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300493)|dotnet/runtime|[nativeaot.SmokeTests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=300493&view=ms.vss-test-web.build-test-results-tab&runId=6086118&resultId=100023)|dotnet/runtime#87290|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|11|11|11|

**Comment by MichalStrehovsky** [comment]
This should have been fixed with #87301 yesterday. Is this an old build? (On my phone now)

**Comment by jkotas** [comment]
Fixed by #87301
